services-manager/src/main/java/uk/gov/gchq/palisade/service/manager/service/ManagedService.java

```diff
@@ -94,7 +93,7 @@ public void setLoggers(final String packageName, final String configuredLevel) t
             Response response = failures.get();
             LOGGER.error("An error occurred while setting logging levels: {}", response);
             String responseBody = Arrays.toString(response.body().asInputStream().readAllBytes());
-            throw new IOException(String.format("Expected /actuator/loggers/%s %s -> 200 OK but instead was %s", packageName, configuredLevel, responseBody));
+            throw new UnsupportedOperationException(String.format("Expected /actuator/loggers/%s %s -> 200 OK but instead was %s", packageName, configuredLevel, responseBody));
```

Comment:
The line where we see the creation of the responseBody uses asInputStream which can throw an IOException, but we are inside a snippet of code meant to build a UnsupportedOperationException that is then meant to be thrown.   So you have a possible outcome that the IOException will be thrown instead of a UnsupportedOperationException. Is that ok here?